--- proptest-0.9.6/Cargo.toml	2020-04-11T20:36:58+00:00
+++ proptest-0.9.6/Cargo.toml	2020-04-15T00:20:20.344722+00:00
@@ -15,7 +15,7 @@
 name = "proptest"
 version = "0.9.6"
 authors = ["Jason Lingle"]
-exclude = ["/gen-*.sh", "/readme-*.md"]
+exclude = ["/gen-*.sh", "/readme-*.md", "/test-persistence-location/"]
 description = "Hypothesis-like property-based testing and shrinking.\n"
 homepage = "https://altsysrq.github.io/proptest-book/proptest/index.html"
 documentation = "https://altsysrq.github.io/rustdoc/proptest/0.9.4/proptest/"
